app/institutions/dashboard/-components/panel/template.hbs

```diff
@@ -0,0 +1,10 @@
+<Panel local-class='panel' as |panel|>
+    <panel.heading @title='{{this.title}}' />
+    <panel.body>
+        {{#if this.institution}}
+            {{yield}}
+        {{else}}
+            <LoadingIndicator data-test-loading-indicator @dark={{true}} />
+        {{/if}}
```

Comment:
This is nice for not needing to repeat the if and the loading indicator, but I'm reasonably certain that this information will ultimately not come from the institution model. Better would be to have an `isLoading` parameter to this component that we can pass in whatever is appropriate (in this case, for now, `@isLoading={{this.model.taskInstance.isRunning}}`).